templates/redirect.html

```diff
@@ -0,0 +1,13 @@
+{% extends "layout.html" %}
+
+{% block htmlhead %}
+  <meta http-equiv="refresh" content="0; URL={{ config.redirect_url }}" />
+  {{ super() }}
+{% endblock htmlhead %}
```

Comment:
Would it make more sense to use the existing `extrahead` block? Or does this need to come first?